contracts/Oracle.sol

```diff
@@ -205,6 +222,18 @@ contract Oracle is Ownable {
         emit DisputerUpdated(_disputer);
     }
 
+    /**
+     * @notice set stable asset price
+     * @dev price should be scaled by 1e8
+     * @param _asset asset address
+     * @param _price price
+     */
+    function setStablePrice(address _asset, uint256 _price) external onlyOwner {
+        stablePrice[_asset] = _price;
+
+        emit StablePriceUpdated(_asset, _price);
+    }
```

Comment:
maybe add in the comments / documentation that this will only be used for USDC / what other assets? That way its clear to other developers who read our documentation :D